apps/etterlatte-saksbehandling-ui/client/src/components/oppgavebenken/typer/oppgavefelter.ts

```diff
@@ -73,11 +73,11 @@ export const initialOppgaveFelter = (): IOppgaveFelter => {
 
     oppgaveType: {
       noekkel: 'oppgaveType',
```

Comment:
Skulle du typesikre accessor og noekkel og?

Reply:
FML react-table-bruken er forvirrende og kompleks og overlapper en del med designsystemet. Laget denne Jira-oppgaven for å se på det: https://jira.adeo.no/browse/EY-1720